kubectl-rook

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Rook Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+# shellcheck source=shellcheck disable=SC1091
+source ceph/ceph_command.sh
```

Comment:
Does this mean the user would have to install a whole bunch of .sh files? It's nice to have them separate during development, but a single script for deployment would keep things much simpler. Maybe we have a build process that combines them? Again, this is fine to resolve in a future PR.

Reply:
@travisn I can keep that in a single file that will be too long but also I'm looking at having exciting krew plugin on how they manage their script. Most of them don't have a big script and if they have one they keep it in a single file.

Reply:
I vote for a single file too, it seems easier just to copy a single file somewhere.

Reply:
okay, I'll move it one file

Reply:
I wonder if we could do a simple file "compile" where. we manage the source code in multiple files but release it as a single file.

Reply:
will follow up PR for this will work? as we want the initial code merge so we can test it with `krew`